keyboards/ashpil/modelm_usbc/rules.mk

```
@@ -0,0 +1,34 @@
+# MCU name
+MCU = at90usb1286
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

```suggestion
BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```

Then we just set this. Much easier than the combo IMO.

---

Oh wow, didn't know that was possible. You're right, that is better. Fixed. Thanks for bearing with me here, I'm new to QMK.